lib/bootloader_setup.pm

```diff
@@ -538,7 +538,7 @@ sub select_bootmenu_more {
     }
     send_key_until_needlematch($tag, get_var('OFW') ? 'up' : 'down', 10, 3);
     # Redirect linuxrc logs to console when booting from menu: "boot linux system"
-    push @params, "linuxrc.log=/dev/$serialdev" if get_var('LINUXRC_BOOT');
+    push @params, "linuxrc.log=/dev/$serialdev linuxrc.debug=4,trace linuxrc.core=/dev/$serialdev" if get_var('LINUXRC_BOOT');
```

Comment:
Would be great if we can extract common part from `https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/bootloader_setup.pm#L400` and this usage.